content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -255,6 +255,7 @@ This section contains important information about installation and deployment or
 
 You must install a pod network add-on so that your pods can communicate with
 each other.
+#Note - This need to be done only on the Control-Plane / on the Master Node
```

Comment:
it can be done on any node that has the kubeconfig credentials.

instead of adding this note here, please change this sentence bellow near the kubectl command:

> You can install a pod network add-on with the following command:

to:

> You can install a pod network add-on with the following command on the control-plane node or a node that has the kubeconfig credentials:

Reply:
done